python/MANIFEST.in

```diff
@@ -17,6 +17,8 @@
 global-exclude *.py[cod] __pycache__ .DS_Store
 recursive-include deps/jars *.jar
 graft deps/bin
+graft deps/data
```

Comment:
Since we limit this to *.data and *.txt later would it make sense to restrict this here as well? That being said I can see an argument that we might have csv or tsv files in the future in which case we'd want to switch the include downstream to be * rather than just txt / data.